mep/people/models.py

```diff
@@ -324,6 +324,13 @@ def viaf(self):
         if self.viaf_id:
             return ViafEntity(self.viaf_id)
 
+    @property
+    def short_name(self):
+        '''Shortened form of name used for locations where space is tight,
+        e.g. breadcrumb navigation'''
+        if self.sort_name:
```

Comment:
`sort_name` can't be null and is a required field, so you don't need this check.

You should handle names like **H. D. (Hilda Doolittle)**, though. I think returning the portion before the ` (` will work.